### PR TITLE
Projects added to Portfolio

### PR DESCRIPTION
**Pull request by SamuelTakyiBoateng**
Index.html (modified to include projects in portfolio)
style.css (modified )
Readme updated

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,5 +45,324 @@ <h2 class="headline__intro">I'm a Software Developer</h2>
...
+    <section>
+      <div class="projects-container">
+        <div class="projects_page1">
+          <h2 class="projects_page1-works">My Recent Work</h2>
+          <hr class="line" />
+          <img
+            class="projects_page1-image"
+            src="images/Img Placeholder.png"
+            alt="image"
+          />
```
Comment (TadesseDev) [inline index.html:57]: - [ ] dear @SamuelTakyiBoateng, as you can see your design margin/padding is a bit odd compared with the Figma design. as front-end engineers we need to be able to clone a template into its identical webpage. Please consider making the appropriate arrangements.

### Your design 
image

### Figma design 
image

File: index.html [hunk 2]
```diff
@@ -45,5 +45,324 @@ <h2 class="headline__intro">I'm a Software Developer</h2>
...
+            </div>
+            
+            <!-- Personal Profile -->
+            <section>
+              <div class="aboutMe">
+                <h2 class="aboutMe_Title">About me</h2>
+                <p class="aboutMe_description">
+                  Hello I’m a software developer! I can help you build a product ,
+                  feature or website Look through some of my work and experience! If you
+                  like what you see and have a project you need coded, don’t hesitate to
```
Comment (TadesseDev) [inline index.html:261]: - [ ] dear @SamuelTakyiBoateng, as you can see your design margin/padding is a bit odd compared with the Figma design. as front-end engineers we need to be able to clone a template into its identical webpage. Please consider making the appropriate arrangements.

### Your design 
image


### Figma design 
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @SamuelTakyiBoateng 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitHub Flow  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (SamuelTakyiBoateng) [reply index.html]: Thank you for the impeccable observation. Proposed changes have been made as advised.
🙏

File: index.html (hunk 2 above)
Comment (SamuelTakyiBoateng) [reply index.html:265]: Thank you for the impeccable scrutiny and comment. The changes effected affect re-organize the code arrangement.
🙏

File: index.html [hunk 3]
```diff
@@ -45,5 +45,324 @@ <h2 class="headline__intro">I'm a Software Developer</h2>
       </ul>
       <div id="background-img" class="headline__background-img"></div>
     </section>
+    <section>
+      <div class="projects-container">
```
[the remark this answers is not in the document]
Comment (SamuelTakyiBoateng) [reply index.html:50]: Thank you. The changes were made and both section have been successful separated.
I really appreciate you advise.

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @SamuelTakyiBoateng 👋, 
giphy
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.
- ✔️ UI design 👍 

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**